sdk/automation/Azure.ResourceManager.Automation/samples/Generated/Samples/Sample_AutomationAccountResource.cs

```diff
@@ -456,7 +456,7 @@ public async Task GetFieldsByType_GetAListOfFieldsOfAGivenTypeAcrossAllAccessibl
         public async Task ConvertGraphRunbookContent_GetGraphicalRawRunbookContentFromGraphicalRunbookJSONObject()
         {
             // Generated from example definition: specification/automation/resource-manager/Microsoft.Automation/stable/2021-06-22/examples/serializeGraphRunbookContent.json
-            // this example is just showing the usage of "ConvertGraphRunbookContent" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "convertGraphRunbookContent" operation, for the dependent resources, they will have to be created separately.
```

Comment:
first character should be lower case: https://github.com/Azure/azure-rest-api-specs/blob/c852a76b96e5c71972cdc62a303665fec7c867b6/specification/automation/resource-manager/Microsoft.Automation/stable/2024-10-23/operations.json#L81